demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1101,6 +1101,45 @@ def is_mapping_fields_command_exist(self) -> bool:
                     return False
         return True
 
+    def are_commands_in_readme(self) -> bool:
+        """
+        Check if commands from the YML file exist in the README file.
+        Returns:
+            True if all commands exist, else False.
+        """
+        valid = True
+        commands_to_ignore = ['get-mapping-fields']
+        dir_path = os.path.dirname(self.file_path)
+
+        if not os.path.exists(os.path.join(dir_path, 'README.md')):
+            return True
+
+        readme_path = os.path.join(dir_path, 'README.md')
+
+        # get README file's content
+        with open(readme_path, 'r') as readme:
+            readme_content = readme.read()
+
+        commands = self.current_file.get("script", {})
+        commands_not_in_yml = []
+
+        # handles scripts
+        if not commands:
+            return True
```

Comment:
why is this needed? If I understood correctly, scripts will not get to `integration.py`.

Reply:
You are right, removed.